src/tools/mediaCapabilitiesProber/probers/_isTypeSupportedWithFeatures_/format.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2015 CANAL+ Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  IAudioConfiguration,
+  IDisplayConfiguration,
+  IOutputProtectionConfiguration,
+  IVideoConfiguration,
+} from "../../types";
+import { findDefaultVideoCodec } from "../defaultCodecsFinder";
+
+const formatConfigForAPI = (
```

Comment:
Why an arrow function?

Reply:
(I actually find it less lisible for top-level function to have arrow-function assignement.

That plus the fact that they are not hoisted, I much prefer regular function style when we are speaking about top-level.